build.bash

```diff
@@ -177,6 +186,7 @@ EOF
 #### Build script start ####
 ############################
 
+trap cleanup_build EXIT ERR
```

Comment:
installing the trap before setting $buildfolder might introduce some problems (actually unmounting `/boot` and `/root`...

Reply:
that's supposed to work, $buildfolder will be evaluated at runtime.
But I've added a check in cleanup_build() for $buildfolder empty just in case